gulpfile.js

```diff
@@ -228,7 +228,7 @@ gulp.task('build', function (done) {
         }))
         .pipe(es.wait(function (err, body) {
             gulp_util.log('Collecting static files');
-            run('./manage.py collectstatic --noinput')
+            run('docker-compose run web python manage.py collectstatic --noinput')
```

Comment:
Occhio che non so se questo deve stare qui.

Reply:
Mi ricordo che nel branch `italia-2.3.13` lo avevamo modificato perché in locale gulp da errore se non lo si avvia con docker. Però effettivamente non so se viene usato in una pipeline di build.
Facciamo che lo rimuovo, torno indietro.
Se serve in locale lo modifico manualmente!